.ci/jenkins/ci.groovy

```diff
@@ -152,23 +181,33 @@ def runDockerContainer(String imageName) {
     """
 }
 
-def prepareEnvironment(List<String> backends) {
+def prepareEnvironment(List<String> backends, String ngraph_branch) {
     String backendsString = backends.join(",")
     sh """
         docker exec ${DOCKER_CONTAINER_NAME} bash -c "${DOCKER_HOME}/${CI_DIR}/prepare_environment.sh \
                                                                             --build-dir=${DOCKER_HOME} \
-                                                                            --backends=${backendsString}"
+                                                                            --backends=${backendsString} \
+                                                                            --ngraph-branch=${ngraph_branch}"
     """
 }
 
 def runToxTests(String backend) {
     String toxEnvVar = "TOX_INSTALL_NGRAPH_FROM=\${NGRAPH_WHL}"
     String backendEnvVar = "NGRAPH_BACKEND=${backend.toUpperCase()}"
-    sh """
-        NGRAPH_WHL=\$(docker exec ${DOCKER_CONTAINER_NAME} find ${DOCKER_HOME}/ngraph/python/dist/ -name 'ngraph*.whl')
-        docker exec -e ${toxEnvVar} -e ${backendEnvVar} -w ${DOCKER_HOME}/ngraph-onnx ${DOCKER_CONTAINER_NAME} \
-            tox -c .
-    """
+    String libraryVar = (backend == "ie") ? "LD_LIBRARY_PATH=${DOCKER_HOME}/dldt_dist/deployment_tools/inference_engine/external/tbb/lib:${DOCKER_HOME}/dldt_dist/deployment_tools/inference_engine/lib/intel64:${DOCKER_HOME}/dldt_dist/deployment_tools/inference_engine/external/mkltiny_lnx/lib:${DOCKER_HOME}/dldt_dist/deployment_tools/ngraph/lib" : "LD_LIBRARY_PATH="
+   
+
+    if (backend == "ie") {
```

Comment:
This doesn't execute unless there's "ie" in `backends` list here (same file, line 23):
```
BACKEND_SKU_CONFIGURATIONS = [
        [ sku : "skx", backends : ["cpu", "interpreter"] ],
        [ sku : "clx", backends : ["cpu", "interpreter"] ],
        [ sku : "bdw", backends : ["cpu", "interpreter"] ]
```

Reply:
Nevermind, I understand this PR just adds possibility to run tests on IE backend that will be switched on later.